ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -68,87 +68,95 @@ export class ConfigurationsComponent implements OnInit {
       .subscribe(
-        data => {
-          this.formData = data;
-          this.toastr.success("Success!");
-        },
-        error => {
-          switch (error.status) {
-            case 404: {
-              this.toastr.error("File not found!", "An error occured!");
-              break;
-            }
-            case 400: {
-              this.toastr.error(
-                "Check your field inputs",
-                "An error occurred!"
-              );
-              break;
-            }
-            case 500: {
-              this.toastr.error("An error has occurred!", "Check the logs!");
-              break;
-            }
-            default: {
-              this.toastr.error(
-                "Your data was not retrieved!",
-                "An error occurred!"
-              );
-            }
+      data => {
+        this.formData = data;
+        this.toastr.success("Success!");
+      },
+      error => {
+        switch (error.status) {
+          case 404: {
+            this.toastr.error("File not found!", "An error occured!");
+            break;
+          }
+          case 400: {
+            this.toastr.error(
+              "Check your field inputs",
+              "An error occurred!"
+            );
+            break;
+          }
+          case 500: {
+            this.toastr.error("An error has occurred!", "Check the logs!");
+            break;
+          }
+          default: {
+            this.toastr.error(
+              "Your data was not retrieved!",
+              "An error occurred!"
+            );
           }
         }
+      }
       );
   }
   onUpdate(configData) {
     this.automatedUIServices
       .putConfig$(this.formData, this.configPath, this.xmlFileName)
       .subscribe(
-        data => {
-          this.toastr.success("Success!");
-        },
-        error => {
-          switch (error.status) {
-            case 404: {
-              this.toastr.error("File not found", "An error occured!");
-              break;
-            }
-            case 400: {
-              this.toastr.error(
-                "File must be specified!",
-                "An error occurred!"
-              );
-              break;
-            }
-            case 500: {
-              this.toastr.error("Internal server error!");
-              break;
-            }
-            default: {
-              this.toastr.error("File was not updated!", "An error occurred!");
-            }
+      data => {
+        this.toastr.success("Success!");
+      },
+      error => {
+        switch (error.status) {
+          case 404: {
+            this.toastr.error("File not found", "An error occured!");
+            break;
+          }
+          case 400: {
+            this.toastr.error(
+              "File must be specified!",
+              "An error occurred!"
+            );
+            break;
+          }
+          case 500: {
+            this.toastr.error("Internal server error!");
```

Comment:
keep these consistent

e.g. above, you have used "this.toastr.error("An error has occurred!", "Check the logs!");"